[docs] CDC outline and hierarchy (#17030)
yugabyte/yugabyte-db: docs/content/preview/explore/change-data-capture/cdc-tutorials/cdc-aws-msk.md

```

If you have a multi-node YugabyteDB setup, you need to provide a comma-separated list of **host:port** values of both the leader and the follower nodes as the `master_addresses` argument.

If successful, the command returns the CDC stream ID:

```output
CDC Stream ID: 90fe97d59a504bb6acbfd6a940
```

For more information on CDC commands, refer to [Change data capture commands](../../../../admin/yb-admin/#change-data-capture-cdc-commands).

### Configure the AWS Security Group

Create a Security Group with inbound and outbound rules configured to ensure access to the MSK cluster and YugabyteDB. In this example, enable incoming traffic from all the ports.

![Edit Inbound Rules](/images/explore/cdc/aws_msk_images/edit-inbound-rules.png)

### Upload Debezium connector Jar file to the S3 bucket

Download the YugabyteDB Debezium connector jar from the [repository](https://github.com/yugabyte/debezium-connector-yugabytedb/releases/download/v1.9.5.y.19/debezium-connector-yugabytedb-1.9.5.y.19.jar) and upload it to an S3 bucket.

![Upload to S3](/images/explore/cdc/aws_msk_images/upload-to-s3.png)

### Configure the Amazon MSK cluster

> This example creates an Amazon MSK cluster in same VPC as that of the YugabyteDB cluster. Note that this is a generic configuration, it might differ based your organizational IT policy.

Reviewer: ```suggestion
For this example, create an Amazon MSK cluster in the same VPC as that of the YugabyteDB cluster. Note that this is a generic configuration, and it may differ based on your organizational IT policy.
```

![MSK cluster settings](/images/explore/cdc/aws_msk_images/msk-cluster-settings.png)

> This example creates a cluster with two zones.

Reviewer: ```suggestion
Create a cluster with two zones.
```

> This example creates a cluster with two zones.

Reviewer: ```suggestion
Create a cluster with two zones.
```

![Amazon Brokers](/images/explore/cdc/aws_msk_images/amazon-brokers.png)

> Under the Networking section, select the same VPC and Private subnets as used by the YugabyteDB cluster.

Reviewer: ```suggestion
In the **Networking** section, select the same VPC and Private subnets as used by the YugabyteDB cluster.
```

Choose the security group you created previously.

![Amazon Security Groups](/images/explore/cdc/aws_msk_images/amazon-security-groups.png)

Enable logging on your cluster to help with debugging. This example uses the S3 bucket to store the logs.

![Amazon S3 logging](/images/explore/cdc/aws_msk_images/amazon-s3-logging.png)

> The cluster is now is now configured successfully.

Reviewer: ```suggestion
The cluster is now configured successfully.
```
